src/pages/components/date-picker/usage.mdx

```diff
@@ -107,57 +131,326 @@ Date and time pickers allow users to select a single or a range of dates and tim
+
+#### Selecting a date
+
+A date can be selected by:
+
+- Manually entering a date in the text input field
+- Clicking on a date in the calendar menu
+- Navigating to a date by using the `Arrow` keys and then pressing `Enter`.
+
+#### Next and previous month
+
+A user can navigate between the months in a year by:
+
+- Clicking on `chevron` icons at the top left and right of the calendar
+- Using the `Arrow` keys to move through the into the next or previous month
+
+#### Selecting a year
+
+By default the current date and year will appear in the calendar. To navigate to another year the user can do one of the following:
+
+- Manually typing in the year in the date text field.
+- Clicking on the up and down arrows that appear when you focus or hover on the year input in the calendar.
+- Selecting then typing into the year input.
+
+#### Closing the calendar
+
+The calendar can be closed in one of the following ways:
+
+- Selecting a single date or the end date in a range (will automatically close it)
+- Clicking anywhere outside of calendar menu
+- Removing focus from the picker
+- Pressing `Esc`
+
+#### Selecting a range
 
-The simple date picker provides the user with a text input in which they can input month/day/year. Simple date pickers are typically used when the date is known by the user, such as a birthday or credit card expiration.
+There are several ways in which a range can be selected:
+
+- Manually type the date for the start and end dates in the text field.
+- Once the calendar is open the first date you click will be the start date and second date you click will be the end date.
+- Navigating to the start date by using the `Arrow` keys and pressing `Enter`. Then continue using the `Arrow` keys to navigate to a second date and press `Enter` again.
+
+#### Min and max dates
+
+In order to constrain the possible selectable dates in a calendar, a min and max date may be set. Once set, only the dates that fall within the min/max range will be selectable with the dates outside of the range being disabled.
+
+Use min and max dates to help prevent user error. If a user cannot select dates in the past when scheduling the set a min date to today.
+
+<Row>
+<Column colLg={8}>
+
+![Min and max date example](images/calendar-picker-min-max.png)
+
+</Column>
+</Row>
+
+<Caption>The dates before today are out of range and disabled. </Caption>
 
 ## Time picker
 
-Time pickers provide the user with a text input in which they can input hours/minutes. Additionally, they can be accompanied by an “AM/PM” selection and a time zone selection, which is styled as an [inline select](/components/select/usage).
+Time pickers provide the user with a text field in which they can input hour and minutes. Additionally, they can be accompanied by an AM/PM and a time zone control styled as selects.
+
+The time field format should include the hour and minutes, ie 11:30. It may be localized accommodate the 12hour or 24 hour format.
```

Comment:
```suggestion
The time field format should include the hour and minutes, for example 11:30. It may be localized accommodate the 12-hour or 24-hour format.
```